CONTRIBUTING.md

```diff
@@ -1,64 +1,76 @@
-Contributing to OpenZeppelin Contracts
-=======
+# Contributing Guidelines
 
-We really appreciate and value contributions to OpenZeppelin Contracts. Please take 5' to review the items listed below to make sure that your contributions are merged as soon as possible.
+Thank you for your interest in contributing to Klaytn Contracts. As an open source project, Klaytn Contracts is always open to the developer community and we welcome your contribution to help more developer onboarding resources for the Klaytn developer community. Please read the guideline below and follow it in all interactions with the project.
 
-## Contribution guidelines
+## How to Contribute on Klaytn Docs
 
-Smart contracts manage value and are highly vulnerable to errors and attacks. We have very strict [guidelines], please make sure to review them!
+1. Read this [contributing document](./CONTRIBUTING.md).
+2. Sign [Contributor Licensing Agreement (CLA)](#contributor-license-agreement-cla).
+3. Submit an issue with a proper [labelling](#usage-of-labels).
+4. Please wait until the label changes to `contribution welcome` - otherwise, it is not ready to be worked on.
+5. Only after the label changed to `contribution welcome`, you can start submitting the changes. To avoid any duplicate efforts, it is recommended to update the issue so that other contributors could see someone working on the issue.
+6. Before making a Pull Request (PR), please make sure the suggested content changes are accurate and linked with the corresponding issue reported. After submitting the PR, wait for code review and approval. The reviewer may ask you for additional commits or changes.
+7. All PRs should be made against the `master` branch, once the change has been approved, the PR is merged by the project moderator.
+8. After merging the PR, the pull request will be closed. You can then delete the now obsolete branch.
 
-## Creating Pull Requests (PRs)
+## Types of Contribution
+There are various ways to contribute and participate. Please read the guidelines below regarding the process of each type of contribution.
 
-As a contributor, you are expected to fork this repository, work on your own fork and then submit pull requests. The pull requests will be reviewed and eventually merged into the main repo. See ["Fork-a-Repo"](https://help.github.com/articles/fork-a-repo/) for how this works.
+-   [Issues and Bugs](#issues-and-bugs)
+-   [Feature Requests](#feature-requests)
+-   [Code Contribution](#code-contribution)
```

Comment:
#code-contribution will need to be created

Reply:
added and pushed the code